runtime/lua/vim/_defaults.lua

```diff
@@ -179,7 +179,7 @@ if tty then
   --- already set by the user.
   ---
   --- @param option string Option name
-  --- @param value string Option value
+  --- @param value string|boolean Option value
```

Comment:
Didn't panders already fix this and you included it in the "small fixes" PR.

Reply:
Missed that. Thanks